resource_obmcs_identity_compartment.go

```diff
@@ -15,6 +15,42 @@ import (
 
 // ResourceIdentityCompartment exposes an IdentityCompartment Resource
 func CompartmentResource() *schema.Resource {
+	compartmentSchema := map[string]*schema.Schema{
+		"id": {
+			Type:     schema.TypeString,
+			Computed: true,
+			ForceNew: true,
```

Comment:
Does ForceNew make any difference for ID?

Reply:
Yeah thats a little weird.. not settable field to begin with, results in tf error--removed.